google/cloud/forseti/services/inventory/base/cai_gcp_client.py

```diff
@@ -1148,22 +1148,25 @@ def iter_crm_projects(self, parent_type, parent_id):
         for project in resources:
             yield project
 
-    def iter_crm_organization_access_policies(self, org_id):
-        """Iterate access policies in an organization from Cloud Asset data.
+    def fetch_crm_organization_access_policy(self, org_id):
```

Comment:
This change is not related to org policy.

Reply:
Removed